src/content/en/tools/chrome-user-experience-report/getting-started.md

```diff
@@ -0,0 +1,216 @@
+project_path: /web/tools/_project.yaml
+book_path: /web/tools/_book.yaml
+
+{# wf_updated_on: 2017-10-23 #}
+{# wf_published_on: 2017-10-23 #}
+
+# Chrome User Experience Report {: .page-title }
+
+## Getting started
```

Comment:
Nit: DevSite isn't always consistent in how it creates anchors, recommend you force them... For example, use:
```
## Getting started {: #getting-started }
```